docs/Consensus-Protocols/Comparing-PoA.md

```diff
@@ -20,24 +20,33 @@ and add blocks to the blockchain. Nodes are added to or removed from the signer/
 Properties to consider when comparing Clique and IBFT 2.0 are: 
 
 * Immediate finality 
+* Minimum number of validators 
 * Liveness 
 * Speed 
-* Minimum number of validators 
 
 ### Immediate Finality 
 
 IBFT 2.0 has immediate finality. When using IBFT 2.0 there are no forks and all valid blocks are included in the main chain.
 
 Clique does not have immediate finality. Implementations using Clique must be aware of forks and chain reorganizations occurring. 
 
+### Minimum Number of Validators 
+
+IBFT 2.0 requires 4 validators to be Byzantine fault tolerant. 
```

Comment:
Given this is the comparison file - should something be said about Clique's requirement? eg. Clique is able to operate with a single signer, however this offers no redundancy in the event of a node failure?

Reply:
Good point. Fixed.